site.json

```diff
@@ -36,7 +36,8 @@
       "startDate": "2022-12-16T17:00:00.000Z",
       "endDate": "2022-12-30T16:00:00.000Z",
       "text": "Node.js assessment of OpenSSL 3.0.7 security advisory",
-      "link": "https://nodejs.org/en/blog/vulnerability/openssl-fixes-in-regular-releases-dec2022/"
+      "link": "https://nodejs.org/en/blog/vulnerability/openssl-fixes-in-regular-releases-dec2022/",
+      "bannerBtnText": "Read more"
```

Comment:
"Read more" links are usually an accessibility smell. Why is this needed vs the old link-only version?

Reply:
The original component in `nodejs.dev` had two versions.

One with a Banner Text + a button, which was generally used for announcements, which looks like this.

![image](https://user-images.githubusercontent.com/54291836/230210524-df3f506e-bb68-4afb-9d92-5c8a8eaba3fb.png)

And one like the classic one, where one can directly give an html, e.g., `<p>Banner Text</p>` and a link, and it will look like this.
![image](https://user-images.githubusercontent.com/54291836/230210609-de5656ba-d834-4ef2-9690-f2d7a18f0cce.png)

Both versions are here. However the addition of `bannerBtnText` in `site.json` will not affect anything on the website, as the added component is not being used anywhere as of now. I added it here just for reference.

For the new banner component, the input interface looks something like this:

```ts
{
  text: string;
  link: string;
  bannerBtnText: string;
} | {
  html: string;
  link: string;
}
```

Reply:
I don't think we should add `bannerBtnText` to `site.json`. It should probably be on `i18n/locales/en.json` as "components.common.banner.button.text"

Reply:
The data for banner is present in the `site.json` only, and as mentioned above, the buttonText was previously configurable according to the text in banner. For example, there can be an announcement for a youtube video, so the button should say something like `Watch Now`, at-least that was the original purpose of this configuration IMO, I could be wrong though.

I think it makes sense to keep all the configurable data for banner at the same place, but happy to change it if we are fixed on keeping the button text as `Read More` 🤔 

Btw we should also make the banner data localised, but I am guessing that we can tackle that in a separate issue/pr

Reply:
Sorry, but I have to disagree. You should 'at least' if you're adding this new feature to make `bannerBtnText` a i18n string that fallbacks to the content of `bannerBtnText` if there's no i18n key for that banner button :)

Reply:
> Btw we should also make the banner data localised, but I am guessing that we can tackle that in a separate issue/pr

I would rather not delegate such a trivial and important change to a separate PR.

Reply:
@ovflowd The thing is that these tasks have been dealing with migrating the components. The migrated component here takes the bannerData as props, and will be used in the implementation part of the component. What I was thinking that maybe I can remove the bannerBtnText from site.json altogether, and when we will start implementing the components in the pages, then we can think of a way to implement new format of website banner. As other components that are being migrated from nodejs.dev also will require changes in the original configuration when we will try to use them

Reply:
If the task is just about migrating the component, you shouldn't add `bannerBtnText` as this deviates from the migration. If you're not open to considering other angles here, then please remove this change, and open another PR after this one gets merged so we discuss that change.

I won't approve of a change that intentionally introduces a point of failure that "will" someday be addressed.

Reply:
Agreed, my bad as well. The task should have been focused on migrating the component only. I'll remove the entry in site.json in a few hours and then we can merge it. 

Should I create a new interface as well for the new Banner component, or adding the btnText in old interface will work?

Reply:
I guess the appropriate solution for now is a fixed banner button text "Read more" that is a i18n translation key. So no need for a bannerBtnText on the interface or something.

I know your concern is about allowing the user to change the text of that button on a per-banner basis, we can address that in a future PR 🙃

Reply:
Yeah that can work 👍🏼